smart-contract/assembly/contracts/internals/const.ts

```diff
@@ -6,3 +6,5 @@ export const CHUNK_NB_TAG = k.nextKey(); // 2 as u8
 export const METADATA_TAG = k.nextKey(); // 3 as u8
 export const MAX_STORAGE_VALUE_SIZE = k.nextKey(); // 4 as u8
 export const FILES_PATH_LIST = new ConstantManager<Array<string>, u8>(k); // k will be 5 as u8
+export const IMMUTABLE = new ConstantManager<bool, u8>(k); // k will be 6 as u8
+export const DEWEB_VERSION = new ConstantManager<u32, u8>(k); // k will be 7 as u8
```

Comment:
I was more thinking about something like a `\007DEWEB_VERSION`
So we have a byte at the beginning, and then the variable name. So when looking into the ledger, we can check if this key exists. If so, we know it's a website. 

But if we just use 7, a lot of SC might use it, so we will note a lot of SCs as websites while they won't be websites